self-hosted/backup-and-restore/pg-dump-and-restore.md

```diff
@@ -13,15 +13,15 @@ the native PostgreSQL [`pg_dump`][pg_dump] and [`pg_restore`][pg_restore]
 commands. This works even for compressed hypertables, without having to
 decompress the chunks before you begin.
 
-Upgrades between different versions of TimescaleDB can be done in place; you
+Upgrades between different versions of Timescale can be done in place; you
 don't need to backup and restore your data. See
 the [upgrading instructions][timescaledb-upgrade].
 
 <Highlight type="warning">
 If you are using this `pg_dump` backup method regularly, make sure you keep
-track of which versions of PostgreSQL and TimescaleDB you are running. For more
+track of which versions of PostgreSQL and Timescale you are running. For more
 information, see "Versions are mismatched when dumping and restoring a database"
-in the [Troubleshooting section](https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/troubleshooting/).
+in the [Troubleshooting section](https://docs.timescale.com/timescaledb/latest/self-hosted/backup-and-restore/troubleshooting/).
```

Comment:
Hi @charislam! I am not sure about the URL here. Please help.

Reply:
that should be right 👍